src/scss/components/_search.scss

```diff
@@ -6,8 +6,8 @@
   width: auto;
 }
 
-.usa-search .usa-search__submit-text {
-  position: static;
```

Comment:
The `position: static` was inherited from the original brochure site styles, but I can't see that it was actually doing anything. `static` is the default value, and it doesn't appear that this would be overriding anything.